src/commands/codepush/deployment/list.ts

```diff
@@ -66,7 +82,12 @@ export default class CodePushDeploymentListListCommand extends AppCommand {
         )} to see what apps you have access to.`;
         return failure(ErrorCodes.InvalidParameter, appNotFoundErrorMsg);
       } else {
-        return failure(ErrorCodes.Exception, "Failed to get list of deployments for the app");
+        let message = "Failed to get list of deployments for the app";
+        if (error.statusCode === 429) {
+          message =
+            "There are too many request please try disabling fetching the metrics for each deployment by using -s | --skipFetchingDeploymentMetrics flags";
```

Comment:
```ts
message =
            "There are too many request please try disabling fetching the metrics for each deployment by using -s | --skipFetchingDeploymentMetrics flags";
```

My suggestion:
```ts
```ts
message = message +  ". Too many requests. Try disabling metrics request for each deployment by using -s | --skipFetchingDeploymentMetrics flag.";
```

Reply:
Make sense so changed to new message